### PR TITLE
update to finish rspec

### PR DESCRIPTION
**Pull request by Rostammahabadi**

### CODE REVIEW

File: lib/game_team.rb [hunk 1]
```diff
@@ -118,13 +118,13 @@ def self.worst_offense
...
+    game_team_with_max = total_game_teams_per_team_id.max_by{|game_team| game_team.goals}
     return game_team_with_max.goals
   end
 
   def self.fewest_goals_scored(team_id)
     total_game_teams_per_team_id = find_by_team(team_id)
-    game_team_with_min = total_game_teams_per_team_id.min{|game_team| game_team.goals}
+    game_team_with_min = total_game_teams_per_team_id.min_by{|game_team| game_team.goals}
     return game_team_with_min.goals
   end
```
Comment (perryr16) [inline lib/game_team.rb:129]: easy

**Review by perryr16 — COMMENTED** [review]
approved -rp

**Comment by foleymichelle9** [comment]
Looks great! Approved

**Comment by stellakunzang** [comment]
excellent!

**Pull request merged**